src/components/common/Actions.vue

```diff
@@ -181,6 +181,8 @@ export default class Actions extends Mixins(DateMixin, FilingTemplateMixin, Name
   protected async onClickSaveResume (): Promise<void> {
     // prevent double saving
     if (this.isBusySaving) return
+    // If Save and Resume is successful setIsFilingPaying should't be reset to false,
+    // this prevent buttons from being re-enabled if the page is slow to redirect.
```

Comment:
This should be on line 202, right?

Reply:
I figured I should say it where isSavingResuming is set so anyone reading the code sees isSavingResuming is set to true vs if they see a comment about setSavingResume at the end of the function with no corresponding code.

Reply:
OK, I agree with you. Resolving this conversation.